src/components/OrderList.js

```diff
@@ -0,0 +1,24 @@
+import OrderListHeader from "./OrderListHeader";
+import OrderListItem from "./OrderListItem";
+
+export default function OrderList({ orders }) {
+  return (
+    <div className="order-list">
+      {orders.map((order) => {
+        return (
+          <>
+            <OrderListHeader key={order.id} order={order}></OrderListHeader>
+            {order.orderDetails.map((orderDetail) => {
+              return (
+                <OrderListItem
+                  key={orderDetail.id}
+                  orderDetail={orderDetail}
+                ></OrderListItem>
+              );
+            })}
+          </>
+        );
+      })}
+    </div>
```

Comment:
children이 없다면 바로 태그를 닫아주는게 가독성에 좋아요.
(마이너) element만 그대로 반환할 경우 아래처럼 함수 스코프를 열지않고 바로 반환해줄 수 있어요
```suggestion
    <div className="order-list">
      {orders.map(order => (
        <Fragment key={order.id}>
          <OrderListHeader order={order}></OrderListHeader>
          {order.orderDetails.map(orderDetail => (
            <OrderListItem key={orderDetail.id} orderDetail={orderDetail} />
          ))}
        </Fragment>
      ))}
    </div>
```

Reply:
단일 요소를 반환하는 경우 return을 생략해도 되는군요! 👍🏻 - [a21c6ed](https://github.com/next-step/react-shopping-cart/pull/29/commits/a21c6ed1930b50cdea5d78d272096f16ba5d4eaa)